pages/_app.tsx

```diff
@@ -13,6 +13,20 @@ import { useEffect } from 'react';
 import useLoanFlowStore from '../store/loanFlowStore';
 import useLendFlowStore from '../store/lendFlowStore';
 import useLiquidationFlowStore from '../store/liquidationFlowStore';
+import MoralisV2 from 'moralis';
+
+const initializeMoralis = async () => {
```

Comment:
do we need this?
`MoralisProvider
					appId={process.env.NEXT_PUBLIC_APP_ID as string}
					serverUrl={process.env.NEXT_PUBLIC_SERVER_URL as string}
					initializeOnMount={true}
>`
this should do the trick ?

Reply:
oh I guess it is temporary to make v2 works.
Just add a comment in the code. Remove those codes when fully migrate to V2

Reply:
Yeah I'll remove those. I plan to uninstall v1 once it's complete and run build. I believe every trace of it can be removed that way